### PR TITLE
VACMS 17829 Only show expandable operating statuses for Vet Centers with Operating Status fields

### PR DESCRIPTION
**Pull request by eselkin**
## Summary

- Vet Center Locations page was showing 
- Go to many Vet Center Locations pages and see Mobile section with empty expandable alerts
- Add conditionals to prevent unnecessary components from being added
- Sitewide Facilities

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17829

## Testing done

- Mobile Vet Centers were displaying empty operating status expandable alerts
- Tested locally
- Added testing for operating status fields being present or not

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

<details>
<summary>Before mobile</summary>

</details>

<details>
<summary>Before desktop</summary>

</details>

<details>
<summary>Updated desktop - with Mobile Vet Centers</summary>

</details>

<details>
<summary>Updated desktop - with nearby that have statuses</summary>

</details>

<details>
<summary>Updated mobile - with nearby that have statuses</summary>

</details>

## What areas of the site does it impact?

Vet Center locations pages

## Acceptance criteria

- [x] Does not show Operating Status for Mobile Vet Centers
- [x] Does not show Operating Status for Vet Centers with no data
- [ ] Accessibility review

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

Check tugboat: 

https://web-ihqrv6rby5lxlkn8zqdxf9u7lk1aplcp.demo.cms.va.gov/salt-lake-city-vet-center/locations/

https://web-ihqrv6rby5lxlkn8zqdxf9u7lk1aplcp.demo.cms.va.gov/tacoma-vet-center/locations

### CODE REVIEW

File: src/site/layouts/tests/vet_center/vet_center_locations_operating_status.unit.spec.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/* eslint-disable @department-of-veterans-affairs/axe-check-required */
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/layouts/tests/vet_center/vet_center_locations_operating_status.unit.spec.js:1]: Not a cypress test

**Comment by eselkin** [comment]
@laflannery discussed in https://dsva.slack.com/archives/C0FQSS30V/p1712944072088649 having viewed the pages on the tugboat.

File: src/site/stages/build/drupal/graphql/vetCenterLocations.graphql.js [hunk 2]
```diff
@@ -25,9 +25,11 @@ fragment vetCenterLocationsFragment on NodeVetCenterLocationsList {
       ... on NodeVetCenter {
         title
         entityBundle
+
```
Comment (maxx1128) [inline src/site/stages/build/drupal/graphql/vetCenterLocations.graphql.js:28]: **Question (non-blocking):** why was this blank line added?

**Review by maxx1128 — APPROVED** [review]
One non-blocking comment, the code and the tugboat all look good to me.

File: src/site/stages/build/drupal/graphql/vetCenterLocations.graphql.js (hunk 2 above)
Comment (eselkin) [reply src/site/stages/build/drupal/graphql/vetCenterLocations.graphql.js:28]: Removing

Comment (eselkin) [reply src/site/stages/build/drupal/graphql/vetCenterLocations.graphql.js]: Removed

**Pull request merged**